### PR TITLE
Delete table record from '_schema' if table was created by Presto

### LINKED ISSUE #3082: Mongo connector does not drop table/collection
* `MongoQueryRunner.main()`
* ```
  use mongo.local;
  create table y(a bigint);
  show tables;
  drop table y;
  show tables;
  ```

-- this succeeds, but the `y` table is still there.

### PR DESCRIPTION
**Pull request by emhlbmc**
Fix #3082, if table was created by Presto, it saves metadata in _schema, add '_schema' deleting code is necessary.

### CODE REVIEW

File: presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java [hunk 1]
```diff
@@ -495,6 +497,10 @@ private boolean deleteTableMetadata(SchemaTableName schemaTableName)
 
         MongoDatabase db = client.getDatabase(schemaName);
         if (!collectionExists(db, tableName)) {
+            if (!db.getCollection(META_TABLE_NAME).find(Filters.eq(TABLE_NAME_KEY, tableName)).first().isEmpty()) {
+                DeleteResult result = db.getCollection(META_TABLE_NAME).deleteOne(Filters.eq(TABLE_NAME_KEY, tableName));
+                return result.getDeletedCount() == 1;
+            }
```
Comment (ebyhr) [inline presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java]: How about the following to be consistent?
```java
        if (!collectionExists(db, tableName) &&
                db.getCollection(schemaCollection).find(new Document(TABLE_NAME_KEY, tableName)).first().isEmpty()) {
            return false;
        }
```

Comment (emhlbmc) [reply presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java]: Thanks, your suggestion is better

**Comment by ebyhr** [comment]
@emhlbmc Could you rebase and force-push again to run with the upstream code? I would merge this PR after CI become green.

**Comment by ebyhr** [comment]
@emhlbmc Did you stop working this PR?

**Comment by emhlbmc** [comment]
@ebyhr open another PR here #3234

**Pull request closed without merging**